From ecdf406bd86e3248406b4385b75784758eb5159f Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Thu, 8 Oct 2020 14:22:07 +0700
Subject: [PATCH] arm64: dts: renesas: r8a774e1-hihope-rzg2h: Update DRAM2
 address for ECC

When enable ECC, DRAM2 address of RZ/G2H will change from 0x5 to 0x6, so
that we must change it too.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
index 50b1da0..6b9bbc5 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
@@ -19,9 +19,9 @@
 		reg = <0x0 0x48000000 0x0 0x78000000>;
 	};
 
-	memory@500000000 {
+	memory@600000000 {
 		device_type = "memory";
-		reg = <0x5 0x00000000 0x0 0x80000000>;
+		reg = <0x6 0x00000000 0x0 0x80000000>;
 	};
 
 	reserved-memory {
-- 
1.9.1

